.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/CurrencyUnitValue.cs

```diff
@@ -1,7 +1,19 @@
-﻿namespace Microsoft.Recognizers.Text.NumberWithUnit
+﻿using System.Collections.Generic;
+
+using Microsoft.Recognizers.Text.NumberWithUnit.Utilities;
+
+namespace Microsoft.Recognizers.Text.NumberWithUnit
 {
-    public class CurrencyUnitValue : UnitValue
+    public class CurrencyUnitValue
     {
-        public string IsoCurrency { get; set; } = string.Empty;
```

Comment:
Why was the auto property removed?
Th following format does not raise an warning on my side:
```
public string Number { get; set; } = string.Empty;

public string Unit { get; set; } = string.Empty;

public string IsoCurrency { get; set; } = string.Empty;
```

Reply:
It was a bad merge, fixed